eater/tests/test_eater.py

```diff
@@ -6,15 +6,15 @@ class TestEater(TransactionCase):
     def test_assign_parent_eater(self):
         self.assertTrue(
             self.env.ref("eater.eater2").write(
-                {"parent_eater_id": self.env.ref("eater.eater1")}
+                {"parent_eater_id": self.env.ref("eater.worker1").id}
```

Comment:
it's better practice to create records in the tests and leave demo data for demo.